src/v1.rs

```diff
@@ -130,28 +130,15 @@ impl Party {
     /// Compute this party's share of the group secret key
     pub fn compute_secret(
         &mut self,
-        shares: HashMap<u32, Scalar>,
-        polys: &HashMap<u32, PolyCommitment>,
+        private_shares: HashMap<u32, Scalar>,
+        public_shares: &HashMap<u32, PolyCommitment>,
     ) -> Result<(), DkgError> {
-        if shares.len() != polys.len() {
-            return Err(DkgError::NotEnoughShares(vec![self.id]));
-        }
```

Comment:
Do we need an equivalent check for this?

Reply:
The `MissingPrivateShares` check covers this case, and also provides more information.

Reply:
Hmmm, I think it covers one case, but what if `private_shares` is longer than `public_shares`? I imaging that case is caught elsewhere?

Reply:
I’d argue that we aren’t considering that an error anymore, since it doesn’t stop us from computing the secrets. 

Maybe log a warning?

Reply:
Yeah, maybe.

Are we always going to have a v1 and a v2? Why would someone want to use v1?

Reply:
v1 is exactly frost, just with the Signer interface. It’s conceivable that someone might want to use it for something. It’s much faster than any other frost implementation i benchmarked.